subsystems/drivebase/SwerveDrive.java

```diff
@@ -225,8 +228,13 @@ public void mustangPeriodic() {
             }
         }
         mPoseEstimator.update();
-        SmartDashboard.putNumber("navX heading", getPose().getRotation().getDegrees());
-        SmartDashboard.putNumber("pitch", getPitch());
+        Logger.getInstance().recordOutput(getName()+"/navX Heading Deg", getPose().getRotation().getDegrees());
```

Comment:
since this is library code and it's going to run a lot: java strings aren't the cheapest thing to build. Here we've got 3 strings (the name, the property name and then the concatenation of the two). In addition to the cycles to create those, we're generating more garbage to be collected. Could we not just create constants that we use here?

Reply:
We switched to constants